test-data/unit/diff.test

```diff
@@ -562,3 +562,33 @@ B = Enum('B', 'y')
 [out]
 __main__.B.x
 __main__.B.y
+
+[case testTypedDict]
+from mypy_extensions import TypedDict
+Point = TypedDict('Point', {'x': int, 'y': int})
+p = Point(dict(x=42, y=1337))
+[file next.py]
+from mypy_extensions import TypedDict
+Point = TypedDict('Point', {'x': int, 'y': str})
+p = Point(dict(x=42, y='lurr'))
+[builtins fixtures/dict.pyi]
+[out]
+__main__.Point
+__main__.p
+
+[case testTypedDict2]
```

Comment:
Test also that removing or adding a field triggers a TypedDict type. And test that changing totality triggers the type.